src/adapter/input/post_router.py

```
@@ -58,12 +58,30 @@ def delete_post(post_id:str, authentication_code = Header(None, convert_undersco
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Error in request process: {str(e)}") from e
+
+@router.get(path='/{post_id}',
+            status_code=status.HTTP_200_OK,
+            responses={
+                404: {"description": "Post Not Found"},
+                500: {"description": "Internal Server Error"}
+            },
+            response_model=PostDetailResponse)
+def get_post_detail(post_id:str, service: PostApplicationService = Depends(get_post_application_service)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)